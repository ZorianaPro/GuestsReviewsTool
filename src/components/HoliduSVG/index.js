import React from 'react';

const HoliduSVG = (props) => (
  <svg className="HoliduSVG" {...props}>
    <use xlinkHref="#HoliduSymbol" />
  </svg>
);

export default HoliduSVG;
