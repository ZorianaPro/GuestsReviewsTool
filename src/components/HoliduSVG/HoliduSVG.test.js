import React from 'react';
import { shallow } from 'enzyme';
import HoliduSVG from './';

describe('HoliduSVG', () => {
  it('renders without crashing', () => {
    expect(shallow.bind(shallow, <HoliduSVG />)).not.toThrow();
  });

  it('renders as expected', () => {
    expect(shallow(<HoliduSVG />)).toMatchSnapshot();
  });
});
