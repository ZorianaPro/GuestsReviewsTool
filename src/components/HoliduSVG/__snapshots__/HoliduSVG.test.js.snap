// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HoliduSVG renders as expected 1`] = `
<svg
  className="HoliduSVG"
>
  <use
    xlinkHref="#HoliduSymbol"
  />
</svg>
`;
